src/components/accounts/RecoverAccountSeedPhraseForm.js

```diff
@@ -10,9 +10,25 @@ const RecoverAccountSeedPhraseForm = ({
     isLegit,
     handleChange,
     seedPhrase,
+    accountId,
     requestStatus
 }) => (
         <>
+            <h4><Translate id='recoverSeedPhrase.accountIdInput.title' /></h4>
+            <Translate>
+                {({ translate }) => (
+                    <Input
```

Comment:
This has wrong capitalization setting on iOS (starts from capital letter, which results in invalid account name).

Reply:
Maybe makes sense to use same control as for receiver in Send Money? So that it also checks if account exists.

Reply:
Great catch, I've added `autoCapitalize='off'`. It seems more common to not validate partial information when 'logging in', so I skipped it here too. For example, when you log in to apps, it normally doesn't validate username as you enter it, but only validates username and password together. It's probably not a big deal, but seems weird IMO to do it here. Same if you're trying to reset your password. No app tells you it's a valid email as you're typing it in. I know that you can still check by trying to create an account with that email etc. It's just a common pattern.